Cargo.lock

```diff
@@ -726,7 +726,7 @@ checksum = "ac74c624d6b2d21f425f752262f42188365d7b8ff1aff74c82e45136510a4857"
 
 [[package]]
 name = "prio"
-version = "0.8.3"
+version = "0.9.0"
```

Comment:
Don't we need to cut the new version prior to changing the version? Does this change need to be coordinated with a  GitHub action? cc/ @tgeoghegan for guidance here.

Reply:
prio 0.8.3 has already been published. The `publish-crate` action runs upon publishing a release, and invokes `cargo publish`, so we need to update the manifest prior to creating the git tag. I figured I may as well update the crate version well in advance, to avoid potential confusion when working with unreleased code.